workflow/tests/hpxml_translator_test.rb

```diff
@@ -1016,9 +1020,13 @@ def _verify_outputs(rundir, hpxml_path, results, hpxml)
     end
 
     # HVAC Load Fractions
-    if (not hpxml_path.include? 'location-miami') && (not hpxml_path.include? 'location-honolulu') && (not hpxml_path.include? 'location-phoenix')
+    if (not hpxml_path.include? 'location-miami') && (not hpxml_path.include? 'location-honolulu') && (not hpxml_path.include? 'location-phoenix') && (not hpxml_path.include? 'power-outage-year-round')
```

Comment:
Lots of references to year-round here and below; can they be reverted?

Reply:
Reverted.